Phone book/main.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import "fmt"
+
+func List(){
+	file, err := os.Open("phone book.json")
+	if err != nil {
+  		log.Fatal(err)
+	}
+	defer file.Close()
+
+	s := bufio.NewScanner(f)
+	for s.Scan() {
+  		fmt.Println(s.Text())
+	}
+	if err := s.Err(); err != nil {
+  		log.Fatal(err)
+	}
+}
+
+func Add(){
+	file, err := os.Open("phone book.json")
+	if err != nil {
+  		log.Fatal(err)
+	}
+	defer file.Close()
+
+	s := bufio.NewScanner(f)
+	for s.Scan() {
+  		s.Text()
+	} 
+	if err := s.Err(); err != nil {
+  		log.Fatal(err)
+	}
+
+	type phone struct {
+		name string
+		surname string
+		number int
+	}
+	
+	gg := []float64{s}
+	fmt.Println("Enter name,surname and phone number")
+	fmt.Scan(&phone)
+	add := append(gg ,phone)
+	fmt.Printl(add)
```

Comment:
You aren't adding the structure to a slice of structures and JSON-encoding them to a file